madgiraffestudios/src/App.scss

```diff
@@ -1,6 +1,11 @@
 @import "~bootstrap/scss/bootstrap";
 $font-size-base: 0.9rem !default; 
 .navbar{
-font-size: 20px;
-font-family: 'Poppins',sans-serif;
+   font-size: 20px;
+   font-family: 'Poppins',sans-serif;
```

Comment:
- [ ] this css shouldn't be in navbar class since the font-family will be for the entire project. Please make it global

Reply:
will be addressed in separate issue